### PR TITLE
Adding OM-5 camera (by OM Digital Solutions)

### PR DESCRIPTION
**Pull request by DE-cr**
notes:
- this adds model="OM-5" to https://github.com/darktable-org/rawspeed/pull/357 (which clears the path in adding make="OM Digital Solutions")
- camera metrics copied from Olympus E-M1MarkIII, from which the OM-5 inherits its sensor
- tested with RAW files from https://www.dpreview.com/sample-galleries/1430250048/om-system-om-5-sample-gallery-dpreview-tv (both regular 20 MP files and 50 MP handheld high res)
- having this included with darktable 4.2 would be such a great Christmas gift, to go along with that new camera! :)

### CODE REVIEW

File: data/cameras.xml [hunk 1]
```diff
@@ -7387,6 +7387,18 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
+	<Camera make="OM Digital Solutions" model="OM-5">
```
Comment (kmilos) [inline data/cameras.xml]: > camera metrics copied from Olympus E-M1MarkIII, from which the OM-5 inherits its sensor

Add a comment this is copy of Olympus E-M5 Mark III data

**Comment by kmilos** [comment]
Probably needs samples in RPU before being included, see https://github.com/darktable-org/rawspeed/pull/357#discussion_r1018771751

File: data/cameras.xml (hunk 1 above)
Comment (DE-cr) [reply data/cameras.xml]: Actually, it's a copy of Olympus E-M1MarkIII data.
I didn't notice a way to add comments in the xml, sorry.

Comment (kmilos) [reply data/cameras.xml]: > Actually, it's a copy of Olympus E-M1MarkIII data.

I thought the DP Review said this was basically a rebranded E-M5MarkIII... In any case, it doesn't matter as they both seem to have the same sensor. Black level is ISO sensitive and doesn't matter much here (it is 256 on the DPR ISO 80 sample), and is read dynamically from metadata anyway for ORFs.

> I didn't notice a way to add comments in the xml, sorry.

There are quite a few examples of the "copied from" in the file already...

Comment (DE-cr) [reply data/cameras.xml]: I'm sorry, but I cannot find such comments in `cameras.xml`.
(I have seen them in `*.cpp`, though, but that's not part of the change here.)

E-M1MarkIII seems to have inherited its sensor from the E-M5MarkIII, but there may (or may not) have been some (minor) improvements along the way. I've read in OM-5 reviews that image quality is improved over E-M5MarkIII.

Comment (kmilos) [reply data/cameras.xml]: > I'm sorry, but I cannot find such comments in `cameras.xml`.

True! Looks like they've been blown away by the script re-generating this file, but [I remember seeing them in the (distant) past](https://github.com/darktable-org/rawspeed/blob/9e78948f1a633c343d7a5af45d180ab8fa16e9bc/data/cameras.xml#L5858)... Never mind then, as they will probably again get removed by a script later anyway.

**Comment by DE-cr** [comment]
I've just uploaded a full set of sample raw files for OM-5 to https://raw.pixls.us

**Comment by DE-cr** [comment]
> I've just uploaded a full set of sample raw files for OM-5 to https://raw.pixls.us

...and I've since used this pr in my custom version of darktable 4.0.1 for plenty of raw files from my OM-5, both regular 20 MP pictures, 50 MP handheld hi-res and 80 MP tripos hi-res. Works like a charm (as expected).

**Comment by LebedevRI** [comment]
I accidentally posted this in the PR it seems. Moving here from https://github.com/darktable-org/rawspeed/pull/357#issuecomment-1407538283:

So i was going to merge this, but i'm not going to.
I'm not sure just how exactly else we can reword the message on https://raw.pixls.us/.
NO COLOR CHARTS.

Please contribute the same 5-sample set that is currently available there for that camera,
but of something else, like daylight landscape. Then i'll merge this.

**Comment by LebedevRI** [comment]
@DE-cr thank you for your contribution!

**Comment by DE-cr** [comment]
Sorry, I was too eager to get support for my new camera in darktable's 2022 "Christmas" release that I did the test shots immediately after receiving the OM-5, without carefully reading the instructions. I'll upload a different set.

**Comment by DE-cr** [comment]
@LebedevRI I've now uploaded a new set of raw files for the OM-5 that don't show a color target. Please have the old OM-5 sample files (upload from 2022-11-16) removed from pixls.us

**Comment by LebedevRI** [comment]
Thanks!
Could you please also rebase this, and ideally squash all commits into one?

**Comment by LebedevRI** [comment]
I should have just waited i guess. Please, it's best to use a separate branch for pull requests,
it gets unnessesairly messy otherwise. I've merged this locally in 18f1e185ef84dbae1c353ae6a3cd9a298b95847f.

@DE-cr Thank you for the contribution!

**Comment by DE-cr** [comment]
> Thanks! Could you please also rebase this, and ideally squash all commits into one?

I'm sorry, should I still do anything here, @LebedevRI ?

**Comment by LebedevRI** [comment]
> > Thanks! Could you please also rebase this, and ideally squash all commits into one?
> 
> I'm sorry

That was my fault.

> should I still do anything here, @LebedevRI ?

No. In general, after things are merged, it becomes my problem,
and original contributor can't really be asked for anything more.

**Pull request closed without merging**